src/lamp_py/ingestion/convert_gtfs_rt.py

```diff
@@ -99,19 +106,21 @@ def convert(self) -> None:
         table_count = 0
         try:
             for table in self.process_files():
-                self.write_table(table)
+                if table.num_rows == 0:
```

Comment:
it feels like this function could just work on a max number of files rather than a specific number of hours.

i guess thats all going to go away when we switch to daily archive files and if we want to handle that in that PR thats fine with me.